workshop-report.md

```diff
@@ -0,0 +1,152 @@
+Thomas points out that he has developed a plug-in framework as part of his PoC efforts in Veraison, and it would be possible to take some of the same code into that framework, allowing Veraison to act as a CoSERV proxy and aggregator for NVIDIA RIMs.
+This could become part of a future hackathon, possibly at the forthcoming IETF meeting in Madrid.
+Ned asks about how the signatures were being handled, and Paul responds that there is currently no signature handling or re-signing taking place, because this was only a very brief and constrained prototyping exercise.
+
+### CoSERV API and Implementation
+The group discusses the CoSERV API and its capability to return signed CoRIM from the original source.
+Thomas clarifies that while this functionality is not yet implemented, it is a core requirement to optionally bundle the original data for transparency.
+The conversation then shifts to the timeline for implementing CoSERV support in Veraison, with Thomas explaining that it's currently in a branch but not ready for mainline due to the evolving nature of the specification.
+
+### CDDL Data Model for In-ToTo Attestations
+Henk reports on his morning hackathon activities.
+He provides an overview of In-Toto attestations, explaining their potential use in showing the provenance of reference values in the CoRIM space.
+He shares his exploration of the In-Toto attestation specification, noting its similarities to their existing system and the potential for mapping it to their schema, though some challenges remain due to the complexity of certain predicates.
+
+### Actions and Next Steps
+- Michael to further develop the endorsement creation protocol for physical audits of device locations.
+- Thomas and Michael to determine which working group the endorsement creation protocol document should be submitted to.
+- Kathleen to draft a liaison request for ETSI and ISO documents related to confidential computing standards.
+- Nicolae to send Kathleen the contact information for the head of the ISO work group on confidential computing.
+- Nicolae to provide Kathleen with specific document requests for ETSI and ISO.
+- Confidential computing community to consider ways to align work in ETSI with IETF use cases.
+- Workshop participants to continue discussion on CoSERV and its implementation as a standard for endorsement and reference value conveyance.
+- Paul to add a report summarizing the meeting's discussions and outcomes to the project repository.
+- Paul to continue working on the example proxy plugin for NVIDIA RIMs in the CoSERV implementation.
+- Paul and Thomas to consider adding support for returning original signed CoRIMs as part of the CoSERV API.
+- Paul and Thomas to develop deployment scenarios to clarify different use cases for CoSERV.
+- Thomas to continue developing the CoSERV proof-of-concept in their project branch.
```

Comment:
Could we add a link to the project branch?

Reply:
Thank you, yes, done.